python/interpret_community/mimic/mimic_explainer.py

```diff
@@ -315,6 +316,17 @@ def _get_explain_global_kwargs(self, evaluation_examples=None, include_local=Tru
         if classification:
             kwargs[ExplainParams.CLASSES] = self.classes
         if evaluation_examples is not None:
+            import pandas as pd
+            evaluation_examples_temp = evaluation_examples
+            if isinstance(evaluation_examples_temp, pd.DataFrame):
+                evaluation_examples_temp = evaluation_examples_temp.values
```

Comment:
maybe wrap in a dataset wrapper above, instead of how we do it below, and then it can handle the data-format specific logic so it doesn't have to live here in the explainer?

Reply:
also might just add a get_num_features method to it and have the logic live there

Reply:
yeah also already marked in PR description